wrapper.go

```diff
@@ -130,3 +132,27 @@ func createExporter(printStdout bool, ctx context.Context, logger log.FieldLogge
 	}
 	return newExporter(ctx, logger)
 }
+
+func recoverAndCheckFailWriteSpan() {
+	defer recoverWithLogs()
+	dirEntries, err := os.ReadDir(SPANS_DIR)
+	if err != nil {
+		logger.WithError(err).Error("failed to read spans dir")
+	}
+	found := false
+	for _, file := range dirEntries {
+		if !file.IsDir() {
+			if strings.Contains(file.Name(), "_end") {
+				found = true
+				break
+			}
+		}
+	}
+	if !found {
+		endFile, err := os.Create(filepath.Join(SPANS_DIR, "balagan_end"))
+		if err != nil {
+			logger.WithError(err).Error("failed to create file _end in spans dir lambda will timeout :/")
```

Comment:
The lambda won't reach timeout, the extension has a mechanism that understands when timeout is close.
```suggestion
			logger.WithError(err).Error("failed to create file _end")
```